cookbooks/cdo-repository/recipes/default.rb

```diff
@@ -12,10 +12,19 @@
 end
 
 git "/home/#{node[:current_user]}/#{node.chef_environment}" do
-  repository 'git@github.com:code-dot-org/code-dot-org.git'
+  if node['cdo-github-access'] && node['cdo-github-access']['id_rsa'] != ''
+    repository 'git@github.com:code-dot-org/code-dot-org.git'
+  else
+    repository 'https://github.com/code-dot-org/code-dot-org.git'
+  end
 
+  depth 5
   # Sync to the production or staging branch as appropriate.
-  branch = (node.chef_environment == 'adhoc') ? 'staging' : node.chef_environment
+  if node.chef_environment == 'adhoc'
```

Comment:
Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.